.vib/alertmanager/goss/alertmanager.yaml

```diff
@@ -0,0 +1,9 @@
+command:
+  check-app-version:
+    exec: alertmanager --version | grep -o -m 1 "[0-9]*\.[0-9]*\.[0-9]"
+    exit-status: 0
+    stdout:
+      - {{ .Env.APP_VERSION }}
+  check-binary-run:
+    exec: amtool --help
+    exit-status: 0
```

Comment:
If I'm not mistaken, we are not running `--help` command for binaries other that the primary one. @FraPazGal @joancafom may help clarify this.

Reply:
Yep, you are right :)